python/paddle/fluid/contrib/slim/quantization/quantization_pass.py

```diff
@@ -625,9 +630,8 @@ def __init__(self,
         self._activation_bits = activation_bits
         self._weight_quantize_type = weight_quantize_type
         self._quantizable_ops = quantizable_op_type
-        supported_quantizable_ops = ['conv2d', 'depthwise_conv2d', 'mul']
         for op in self._quantizable_ops:
-            assert op in supported_quantizable_ops, \
+            assert op in QuantizationTransformPass.supported_quantizable_op_type, \
```

Comment:
`assert op in QuantizationFreezePass.supported_quantizable_op_type, \`

Reply:
> `assert op in QuantizationFreezePass.supported_quantizable_op_type, \`

Done